### PR TITLE
✨ Added the Go to Top Button

### PR DESCRIPTION
**Pull request by th3c0d3br34ker**

### CODE REVIEW

**Comment by th3c0d3br34ker** [comment]
@ashutosh1919 have a look, I liked this icon for top button. If you want any changes do tell me.

**Comment by danielmarostica** [comment]
I like the icon but I'd rather use a smaller icon with less opacity, like mine (https://danielmarostica.github.io)

**Comment by ashutosh1919** [comment]
@th3c0d3br34ker , I think I am agree with @danielmarostica's opinion. We should use arrow symbol for more legibility and avoid confusion. Good work overall. :)

**Comment by th3c0d3br34ker** [comment]
@danielmarostica @ashutosh1919 check it out

**Comment by ashutosh1919** [comment]
@th3c0d3br34ker , I have forcefully overwritten the files in order to prettify. Please synchronize the branch and push the changes. I will merge it as soon as you do it.

**Comment by ashutosh1919** [comment]
@th3c0d3br34ker , I think there still is a problem. See below image. The problem may be because of `i` inside `button`. Please take a look at [this code](https://github.com/ashutosh1919/masterPortfolio/blob/master/src/components/button/Button.js) and change the code. also check the deploy preview [here](https://deploy-preview-42--ashutosh1919portfolio.netlify.app/) after changing code.

Screenshot 2020-08-03 at 6 29 46 PM

File: src/components/topButton/TopButton.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        border: `solid 1px ${theme.text}`,
+      }}
+      title="Go up"
+      onMouseEnter={(event) => onMouseEnter(event, theme.text, theme.body)}
+      onMouseLeave={(event) => onMouseLeave(event, theme.body, theme.text)}
+    >
+      <i
+        class="fas fa-arrow-up"
+        aria-hidden="true"
+        onMouseEnter={(event) => onMouseEnter(event, theme.text, theme.body)}
```
Comment (ashutosh1919) [inline src/components/topButton/TopButton.js:53]:
```suggestion
style={{backgroundColor: "transparent"}}
```

File: src/components/topButton/TopButton.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        onMouseLeave={(event) => onMouseLeave(event, theme.body, theme.text)}
```
Comment (ashutosh1919) [inline src/components/topButton/TopButton.js:54]:
```suggestion
```

**Comment by ashutosh1919** [comment]
@th3c0d3br34ker , still not fixed. Please give ideas

**Comment by th3c0d3br34ker** [comment]
I figured out another way, where I define for both the elements.

```
    /* For the button */
    const topButton = document.getElementById("topButton");
    topButton.style.color = color;
    topButton.style.backgroundColor = bgColor;

    /* For arrow icon */
    const arrow = document.getElementById("arrow");
    arrow.style.color = color;
    arrow.style.backgroundColor = bgColor;
```

**Comment by ashutosh1919** [comment]
> I figured out another way, where I define for both the elements.
> 
> ```
>     /* For the button */
>     const topButton = document.getElementById("topButton");
>     topButton.style.color = color;
>     topButton.style.backgroundColor = bgColor;
> 
>     /* For arrow icon */
>     const arrow = document.getElementById("arrow");
>     arrow.style.color = color;
>     arrow.style.backgroundColor = bgColor;
> ```

Please try out and push

**Comment by ashutosh1919** [comment]
@th3c0d3br34ker please synchronize the branch. i think it's working fine. Good work.

File: src/components/topButton/TopButton.css [hunk 3]
```diff
@@ -0,0 +1,11 @@
+#topButton {
+  visibility: hidden;
+  position: fixed;
+  bottom: 1.2rem;
+  right: 1.5rem;
+  cursor: pointer;
+  padding: 15px;
+  outline: none;
+  border-radius: 15px;
```
Comment (ashutosh1919) [inline src/components/topButton/TopButton.css:9]:
```suggestion
  border-radius: 30px;
```

File: src/components/topButton/TopButton.css [hunk 4]
```diff
@@ -0,0 +1,11 @@
+#topButton {
+  visibility: hidden;
+  position: fixed;
+  bottom: 1.2rem;
+  right: 1.5rem;
+  cursor: pointer;
+  padding: 15px;
+  outline: none;
+  border-radius: 30px;
```
Comment (ashutosh1919) [inline src/components/topButton/TopButton.css:9]:
```suggestion
  border-radius: 1rem;
```

File: src/components/topButton/TopButton.css [hunk 5]
```diff
@@ -0,0 +1,11 @@
+#topButton {
+  visibility: hidden;
+  position: fixed;
+  bottom: 1.2rem;
+  right: 1.5rem;
+  cursor: pointer;
+  padding: 15px;
+  outline: none;
+  border-radius: 1rem;
```
Comment (ashutosh1919) [inline src/components/topButton/TopButton.css:9]:
```suggestion
  border-radius: 1.5rem;
```

File: src/components/topButton/TopButton.css [hunk 6]
```diff
@@ -0,0 +1,11 @@
+#topButton {
+  visibility: hidden;
+  position: fixed;
+  bottom: 1.2rem;
+  right: 1.5rem;
+  cursor: pointer;
+  padding: 15px;
+  outline: none;
+  border-radius: 1.5rem;
```
Comment (ashutosh1919) [inline src/components/topButton/TopButton.css:9]:
```suggestion
  border-radius: 1.75rem;
```

File: src/components/topButton/TopButton.css [hunk 7]
```diff
@@ -0,0 +1,11 @@
+#topButton {
+  visibility: hidden;
+  position: fixed;
+  bottom: 1.2rem;
+  right: 1.5rem;
+  cursor: pointer;
+  padding: 15px;
+  outline: none;
+  border-radius: 1.75rem;
```
Comment (ashutosh1919) [inline src/components/topButton/TopButton.css:9]:
```suggestion
  border-radius: 2rem;
```

**Comment by ashutosh1919** [comment]
LGTM. Adding @th3c0d3br34ker for code and @danielmarostica for design. If you have any problem then please do comment here.

**Comment by ashutosh1919** [comment]
@all-contributors please add @th3c0d3br34ker for Code.

**Comment by ashutosh1919** [comment]
@all-contributors please add @danielmarostica for design.

**Pull request merged**